### PR TITLE
[7.x] Handle ECS-compatible deprecation logs emitted by ES 8.0.0+ (#17728)

### PR DESCRIPTION
**Pull request by ycombinator**
Backports the following commits to 7.x:
 - Handle ECS-compatible deprecation logs emitted by ES 8.0.0+  (#17728)

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -327,6 +327,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Release Google Cloud module as GA. {pull}17511[17511]
 - Update filebeat httpjson input to support pagination via Header and Okta module. {pull}16354[16354]
 - Enhance `elasticsearch/server` fileset to handle ECS-compatible logs emitted by Elasticsearch. {issue}17715[17715] {pull}17714[17714]
+- Enhance `elasticsearch/deprecation` fileset to handle ECS-compatible logs emitted by Elasticsearch. {issue}17715[17715] {pull}17728[17728]
+- Enhance `elasticsearch/slowlog` fileset to handle ECS-compatible logs emitted by Elasticsearch. {issue}17715[17715] {pull}17729[17729]
```
Comment (kaiyan-sheng) [inline CHANGELOG.next.asciidoc:331]: this shouldn't be here?

Comment (ycombinator) [reply CHANGELOG.next.asciidoc:331]: Ugh, you're right. Thanks, fixing!

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: Fixed in b4dde9d05.

**Comment by ycombinator** [comment]
CI failures (Filebeat X-Pack) is unrelated. Merging.

**Pull request merged**